lib/runner.js

```diff
@@ -139,7 +139,7 @@ exports.execute = async function (scripts, options, reporter) {
 
     const experiments = scripts.map((script) => {
 
-        script._executed = true;
+        script._executionScheduled = true;
         return script._root;
     });
```

Comment:
Isn't this redundant now?

Reply:
It's a fair question.  When you invoke scripts with the lab CLI it is definitely redundant since `_executionScheduled` is already marked `true`.  If the script is self-invoked (e.g. `node test/my-test-script.js`) or by using lab as a module then it will be set from `false` to `true`, though it may never be read by lab after that anyway.  Overall I lean towards keeping it for consistency in those latter two situations.

Reply:
But it hasn't been scheduled, it's actively running at this point. `_executionScheduled` just guards the automatic execution when the test code calls `.script({ schedule: true })`, and should not need to be set otherwise.

Reply:
I took a look at this, and it turns out that it's necessary to keep this in order for the test suite to function properly, due to the way scripts are sometimes executed directly within the tests.  I assume that is why this check was here to begin with.  It's just an "internal" concept, but I don't mind the idea that every script that's executed is also marked as having been scheduled, even if it was run directly.